### PR TITLE
Improve Acceptance Tests workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
The temptation!

1. Use `ramsey/composer-install` action
2. Remove debugging
3. Comes with 2 years support 🎉

image

### CODE REVIEW

File: .github/workflows/acceptance-tests.yml [hunk 1]
```diff
@@ -69,31 +52,16 @@ jobs:
...
-      run: |
-        if [ "${{ matrix.wp }}" == "nightly" ]
-        then
-          composer require --dev --update-with-dependencies --prefer-dist roots/wordpress-full="dev-main"
-        else
-          composer require --dev --update-with-dependencies --prefer-dist roots/wordpress-full="~${{ matrix.wp }}.0"
-        fi
+      env:
+        WP_CORE_CONSTRAINT: ${{ matrix.wp == 'nightly' && 'dev-main' || format('~{0}.0', matrix.wp) }}
+      uses: ramsey/composer-install@v2
```
Comment (johnbillion) [inline .github/workflows/acceptance-tests.yml:58]: Why do I need to use a third party action for this?

Comment (szepeviktor) [reply .github/workflows/acceptance-tests.yml:58]: As this action does caching, `composer install` or `composer update`, handles timeout, output and more.
Viktor uses it in dozens of projects.

Comment (szepeviktor) [reply .github/workflows/acceptance-tests.yml:58]: You must know Ben!

Comment (szepeviktor) [reply .github/workflows/acceptance-tests.yml:58]: Alternatively you could have own workflows in _community health files_ https://github.com/ergebnis/.github/tree/main/actions/composer

Comment (szepeviktor) [reply .github/workflows/acceptance-tests.yml:58]: @johnbillion Would you merge something based on `plugin-infrastructure`?

Comment (johnbillion) [reply .github/workflows/acceptance-tests.yml:58]: Not just yet, thanks Viktor. I'm making my way through a list of to-do items as I find the the time and based on the similar but not identical testing setup that I have across my plugins.

Comment (szepeviktor) [reply .github/workflows/acceptance-tests.yml:58]: I'd like to promote Ben's action.

**Comment by johnbillion** [comment]
Closing this off now #110 is in. Thanks Viktor!

**Comment by szepeviktor** [comment]
Glad to contribute to John-projects.

**Pull request closed without merging**